### DISCUSSION TITLE
Noise and clicking over analog out

### DISCUSSION THREAD

**Discussion by KeenVox** [ElectroTechnique/electrotechnique.github.io#34] [Hardware Issues]
Hey everyone,

I just finished with my T-Synth.  Everything works so far and I'm getting audio.
But when I put my headphones in the audio jack (1/4") I've got a constant hum in the background. It chances a little in volume when turning down the volumek knob and changes in pitch when selection different patches. When I put my headphones directly into the jack of the audio board there is no hum. 

I tried connecting it to a guitar combo via aux in. There, after I played one sound, I get a constant clicking sound which doesn't stop. I even get the click, when I make a connection from the audio board to the amp.

Any suggestions, what could be the problem?

**Comment by Electrotechnique** [comment]
How are you powering TSynth? Use something that isn't earthed.

**Reply by KeenVox** [reply]
I used my Pc, a usb mobile socket and a powerbank. There is no difference sadly.

**Comment by Electrotechnique** [comment]
Can I suggest changing from the headphone to the line out on the audio board, like this:
[Line out connection](https://budxbeats.com/upload/video/1617587330_uYp_3.jpg)

**Reply by KeenVox** [reply]
That works, but now I cannot change the volume over the pot anymore. Do I have to change the code?

**Reply by Electrotechnique** [reply]
I spent some time thinking about what the volume pot would actually change the volume of and finally settled on the headphone. Line out and USB Audio should be a fixed level, and the volume adjusted on what it is going into. The code can be changed to alter this and voiceMixerM is probably the most convenient. If you wanted to save the volume level with the patch, there are three SPARE variables available for additional patch parameters.

 voiceMixerM.gain(0, 0.25f);
  voiceMixerM.gain(1, 0.25f);
  voiceMixerM.gain(2, 0.25f);
  voiceMixerM.gain(3, 0.25f);

**Reply by KeenVox** [reply]
I settled on adding an extra Lineout Output to the board. Now I have the headphone out and a lineout with fixed volume for recording and live work. It's on the other end of the board and has no noise. It's still bothering me why the headphone jack has that noise in the background.

**Reply by Electrotechnique** [reply]
The headphone jack has a separate ground. Line out is connected to the common ground.

**Reply by KeenVox** [reply]
Yeah I figured. Nevertheless shouldn't it work quietly too?
